src/System.Text.Formatting/src/System/Text/Parsing/InvariantParser.cs

```diff
@@ -60,6 +61,45 @@ public static bool TryParse(ByteSpan text, FormattingData.Encoding encoding, out
             return true;
         }
 
+        [CLSCompliant(false)]
+        public static bool TryParse(Utf8String text, out uint value, out int bytesConsumed)
```

Comment:
These general purpose parsing APIs should be at some point moved to System.Text.Formatting/Parsing. They are more general purpose than JSON parsing.